packages/browser/src/__tests__/__snapshots__/config-snapshot.test.ts.snap

```diff
@@ -242,11 +242,11 @@ exports[`config snapshot for PostHogConfig 1`] = `
   ],
   "properties_string_max_length": "number",
   "defaults": [
-    "\\"2025-11-08\\"",
+    "\\"2025-11\\"",
     "\\"2025-05-24\\"",
     "\\"unset\\""
   ],
-  "__preview_deferred_init_extensions": [
+  "deferred_init_extensions": [
```

Comment:
Isn't `deferred_extensions_init` a better name? Or `defer_extensions_init`? You're the bri'ish tho, your call